sklearn/model_selection/tests/test_validation.py

```diff
@@ -192,27 +188,28 @@ def fit(self, X, Y=None, sample_weight=None, class_prior=None,
         if X.ndim >= 3 and not self.allow_nd:
             raise ValueError('X cannot be d')
         if sample_weight is not None:
-            assert_true(sample_weight.shape[0] == X.shape[0],
-                        'MockClassifier extra fit_param sample_weight.shape[0]'
-                        ' is {0}, should be {1}'.format(sample_weight.shape[0],
-                                                        X.shape[0]))
+            assert sample_weight.shape[0] == X.shape[0], (
+                'MockClassifier extra fit_param ' 
+                'sample_weight.shape[0]' 
+                ' is {0}, should be {1}'.format(sample_weight.shape[0],
+                                                X.shape[0]))
```

Comment:
Nit (only if you do changes for other reviewers), this could be more nicely written as,
```py
                'sample_weight.shape[0] is {0}, should be {1}'
                .format(sample_weight.shape[0], X.shape[0]))
```